publication-commons/src/test/resources/features/UserAccessRights.feature

```diff
@@ -70,41 +72,41 @@ Feature: User access rights
       | PUBLISHED     | UPDATE  |
 
   @notmvp
-  Scenario: USER users read unpublished publication that is shared with them
-    Given a publication with id "pubID"
-    And the publication "pubID" is NOT published
-    And the owner of the publication "pubID" is the user "theCreator"
-    And "theCreator" has given read access to the publication "pubId" to "theUser"
-    When READ is called on behalf of the user "theUser" for the publication "pubID"
-    Then READ returns the publication "pubID"
+  Scenario: users with role USER read unpublished publication that is shared with them
+    Given a publication
+    And the publication's status is DRAFT
+    And the owner of the publication is not the authenticated user
+    And the owner of the publication has given read access to the authenticated user for this publication
+    When READ is called to read the publication on behalf of the authenticated user
+    Then READ returns the saved version of the publication
```

Comment:
change "saved" to DRAFT - just to be more explicit ?

Reply:
"saved" refers to the instance that is in the database